frontend/src/components/Layout/SwiperSection.vue

```diff
@@ -44,7 +44,6 @@
 </template>
 
 <script setup lang="ts">
-import { Swiper, SwiperSlide } from 'swiper/vue';
```

Comment:
Reviewing on GitHub so probably I'm missing something, but why is this unnecessary? We're using those components in the template

Reply:
Oof, VS Code wasn't pulling me to the uses of it so I thought they were unused types. I'll get these back in!